x/gov/README.md

```diff
@@ -208,6 +212,8 @@ This means that proposals are accepted iff:
 * The proportion of `Yes` votes, excluding `Abstain` votes, at the end of
   the voting period is superior to 1/2.
 
+For expedited proposals, it has a higher minimum threshold as its initial parameter, set at 66.7%.
```

Comment:
nit: can we improve wording here?

`For expedited proposals the threshold is higher, initially set at 66.7%.`

I'm not good at writing, that was my best try 😅

Reply:
Updated